src/services/auth/auth.ts

```diff
@@ -1,13 +1,15 @@
-import { SignInRequest, SignInResponse, SignUpRequest} from '@/types/auth.types'
+import { SignInRequest, SignUpRequest } from '@/types/auth.types'
+
+import { backendApi, proxyApi } from '@/services/api'
 
-const BACKEND_BASE_URL = process.env.NEXT_PUBLIC_BACKEND_BASE_URL
 export const SignUp = async (data: SignUpRequest): Promise<Response> => {
-  return await fetch(`${BACKEND_BASE_URL}/v1/auth/sign-up`, {
-    method: 'POST',
-    headers: {
-      'Content-Type': 'application/json',
-    },
-    body: JSON.stringify(data),
-  })
+  return await backendApi.post(`v1/auth/sign-up`, { json: data }).json()
+}
+
+export const SignIn = async (data: SignInRequest): Promise<Response> => {
+  return await proxyApi.post(`api/auth/sign-in`, { json: data }).json()
+}
+
+export const SignOut = async (): Promise<Response> => {
+  return await proxyApi.post(`api/auth/sign-out`)
```

Comment:
```suggestion 
export const SignUp = async (data: SignUpRequest): Promise<Response> => {
  return await backendApi.post(`/v1/auth/sign-up`, { json: data }).json();
};

export const SignIn = async (data: SignInRequest): Promise<Response> => {
  return await proxyApi.post(`/api/auth/sign-in`, { json: data }).json();
};

export const SignOut = async (): Promise<Response> => {
  return await proxyApi.post(`/api/auth/sign-out`);
};
```

RESTful API 관점에서, 경로 앞에 /를 명시적으로 추가하는게 좋을 것 같습니다